cmd/cluster-agent/app/compliance_cmd.go

```diff
@@ -21,6 +21,8 @@ var (
 )
 
 func init() {
-	complianceCmd.AddCommand(common.CheckCmd(&confPath))
+	confPathArray := make([]string, 1)
```

Comment:
I wonder if I should also modify `cluster-agent` to accept several configuration files provided with `-c` instead of this transformation.

Reply:
nit: confPathArray := []string{confPath}